contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -197,6 +197,11 @@
     "with_usercomments_no_mastermob_usercomments.aaf"
 )
 
+MULTIPLE_TIMECODE_OBJECTS_PATH = os.path.join(
+    SAMPLE_DATA_DIR,
+    "multiple_timecode_objects.aaf"
```

Comment:
Can you triple check to make sure the contents of this AAF are clean to share publicly?

Reply:
I removed any notion of production-related info, only things left are Company ("Pixar Animation Studios") and the tool name used ("aafWrapMov"), which I did not bother to change.